webapp/channels/src/components/channel_info_rhs/channel_info_rhs.tsx

```diff
@@ -156,47 +178,56 @@ const ChannelInfoRhs = ({
                 onClose={actions.closeRightHandSide}
             />
 
-            <TopButtons
-                channelType={channel.type}
-                channelURL={channelURL}
-
-                isFavorite={isFavorite}
-                isMuted={isMuted}
-                isInvitingPeople={isInvitingPeople}
-
-                canAddPeople={canManageMembers}
-
-                actions={{toggleFavorite, toggleMute, addPeople}}
-            />
-
-            <AboutArea
-                channel={channel}
-
-                dmUser={dmUser}
-                gmUsers={gmUsers}
-
-                canEditChannelProperties={canEditChannelProperties}
-
-                actions={{
-                    editChannelHeader,
-                    editChannelPurpose,
-                }}
-            />
-
-            <Divider/>
-
-            <Menu
-                channel={channel}
-                channelStats={channelStats}
-                isArchived={isArchived}
-                actions={{
-                    openNotificationSettings,
-                    showChannelFiles: actions.showChannelFiles,
-                    showPinnedPosts: actions.showPinnedPosts,
-                    showChannelMembers: actions.showChannelMembers,
-                    getChannelStats: actions.getChannelStats,
-                }}
-            />
+            <Scrollbars
+                autoHide={true}
+                autoHideTimeout={500}
+                autoHideDuration={500}
+                renderThumbHorizontal={renderThumbHorizontal}
```

Comment:
We don't need `renderThumbHorizontal` as `renderThumbHorizontal` prop is only used to customize the appearance of the horizontal scrollbar thumb and in this case, we don't have a horizontal scroll.

Reply:
Indeed, it's been removed.